doc/version_history.rst

```diff
@@ -6,6 +6,24 @@
 Version History
 ###############
 
+v6.7.0
```

Comment:
I am willing to go either way. I felt a minor release was justified because I expanded the public API. But the two new public functions are not likely to be of any interest to users of salobj, so a patch release is also a reasonable option.